experiments/AMIP/README.md

```diff
@@ -0,0 +1,50 @@
+## Current AMIP components
+
+### Atmosphere
+
+Dynamical core:
+- Equation: non-hydrostatic and fully compressible
+- Prognostic variables: Density, velocity components, total energy, total specific humidity
+- Spatial discretization: Spectral element in the horizontal, finite difference in the vertical, cubed sphere
+- Time stepping: Implicit-explicit additive Runge–Kutta method
+
+Radiation: A scheme based on RRTM for General circulation model applications—Parallel (RRTMGP) (Pincus et al. 2019)
+
+Convection and turbulence: Diagnostic Eddy-diffusivity Mass-Flux (EDMF) scheme with prognostic turbulent kinetic energy. EDMF is a unified parameterization for turbulence and convection. The grid is decomposed into convective updrafts and the turbulent enrionment. Updraft properties are calculated from mass, momentum and energy conservation of the updraft. Mass and momentum exchange between the updrafts and the environment, and
+turbulent mixing in the environment, are represented with physical closures. Currently, only one updraft is used.
+
+Microphysics: 0-moment scheme, where cloud condensate is removed with a constant timescale
+
+Surface fluxes: A scheme based on Monin-Obukhov similarity theory, with constant roughness lengths over land and ocean
+
+Orographic gravity wave drag: None
+
+Non-orographic gravity wave drag: None
+
+Aerosols and chemistry: None
+
+### Land
+
+Dynamical core:
+- Equation: Following Manabe bucket hydrology scheme (Manabe 1969)
+- Prognostic variables: Temperature, water content, snow water content
+- Spatial discretization: Finite difference, multiple independent columns on the sphere
+- Time stepping: Explicit additive Runge–Kutta method
+
+Land surface albedo: 
+- Bare ground: Prescribed from files
+- Snow: Constant
+
+### Sea ice
+
+Thermodynamics: 0-layer model, with prognostic ice surface temperature and fixed ice thickness
+
+### Coupling
+
+Sequential coupling. Fluxes over a heterogeneous surface are calculated using the averaged surface properties.
+
+## References
+
+[Manabe 1969](https://journals.ametsoc.org/view/journals/mwre/97/11/1520-0493_1969_097_0739_catoc_2_3_co_2.xml): Climate and ocean circulation. I. The atmospheric circulation and the hydrology of the earth's surface.
+
```

Comment:
Can we add references to the EDMF?

Reply:
I added some papers on EDMF. None of them are exactly the same as what we have now, but they are the closest reference before we have a new paper on EDMF.